k8s/nginx/container/Dockerfile

```diff
@@ -7,5 +7,5 @@ RUN apt-get update \
     && apt-get clean
 COPY nginx.conf.template /etc/nginx/nginx.conf
 COPY nginx_entrypoint.bash /
-EXPOSE 80 443 27017
+EXPOSE 80 81 443 444 27017
```

Comment:
Since we use 2 different ports for Websocket and HTTP/API traffic, I have decided to map 80/9984 for the API and 81/9985 for WebSocket, and correspondingly 443/9984 for HTTPS/API and 444/9985 for Websocket Secure.